specification/dimensions/chargefrequency.md

```diff
@@ -0,0 +1,40 @@
+# Charge Frequency
+
+Charge Frequency indicates how often a charge would occur. 
```

Comment:
Noticed you use `will` in other areas.
Also add scenarios:
```suggestion
Charge Frequency indicates how often a charge will occur. Charge Frequency is commonly used for scenarios like forecasting and differentiating between one-time and recurring fees for commitment-based discounts.
```
(There might be a better one to call out than CBD here.)